src/ext/property-panel/settings.ts

```diff
@@ -24,6 +24,100 @@ const getStylingPanelConfig = () => ({
       useGeneral: true,
       defaultValue: [],
       items: {
+        headerSection: {
+          translation: "properties.Header",
+          component: "panel-section",
+          items: {
+            headerFontItem: {
+              component: "items",
+              ref: "components",
+              key: "theme",
+              items: {
+                headerFontWrapper: {
+                  component: "inline-wrapper",
+                  items: {
+                    headerFontSize: {
+                      component: "integer",
+                      ref: "header.fontSize",
+                      translation: "properties.fontSize",
+                      width: 9,
+                      min: 5,
+                      max: 300,
+                      defaultValue(item, data, args) {
+                        const currentTheme = args.theme.current();
+                        return parseInt(currentTheme.object?.pivotTable?.header?.fontSize ?? currentTheme.fontSize, 10);
+                      },
+                      change(data) {
+                        data.header.fontSize = !data.header.fontSize
```

Comment:
If I input a non-numerical value, the behaviour is kind of strange. The value is removed and replace but "undefined" as a placeholder

Reply:
yeah this logic looks weird. It is the same as for table. Maybe the logic worked better with the old styling table?